src/screens/Services/Services.js

```diff
@@ -126,95 +125,116 @@ class ServicesScreen extends React.Component<Props> {
     if (isAltalixAvailable === null) loadAltalixInfo();
   }
 
-  getServices = (): Service[] => {
-    const {
-      navigation,
-      isActiveAccountSmartWallet,
-      isSmartWalletActivated,
-      accounts,
-    } = this.props;
+  navigateToArchanovaSupportedServiceScreen = (route: string) => {
+    const { navigation, isArchanovaWalletActivated } = this.props;
+    if (!isArchanovaWalletActivated) return;
```

Comment:
Is this check correct? It seems that it will redirect the user if he has archanova account regardles whether it's currently active or not.

Reply:
Yes, this check is to determine if Archanova wallet is activated (deployed) before it can use any of service (see "requires activation" label for each service), hope it makes sense.